src/client/components/voyage/StartVoyage.js

```diff
@@ -0,0 +1,215 @@
+import React, { Component } from "react";
+import DateTime from "./DateTime";
+import InputField from "./InputField";
+import Dropdown from "./Dropdown";
+import Checkbox from "./Checkbox";
+
+class StartVoyage extends Component {
+  state = {
+    vessels: [],
+    ports: [],
+    optimisationType: [],
+    departure_position: {},
+    arrival_position: {},
+    vessel_id: 0,
+    ETD: "",
+    ETA: "",
```

Comment:
We are changing this to `target_arrival_time` in the upcoming db migration.

Reply:
Please fix this one as well as etd throughout the PR.

Reply:
Comment incorporated